Testscripts/Linux/fio_jason_parser.sh

```diff
@@ -11,6 +11,17 @@ echo $csv_file
 rm -rf $csv_file
 echo "Iteration,TestType,BlockSize,Threads,Jobs,TotalIOPS,ReadIOPS,MaxOfReadMeanLatency,ReadMaxLatency,ReadBw,WriteIOPS,MaxOfWriteMeanLatency,WriteMaxLatency,WriteBw" > $csv_file_tmp
 
+if [ $DISTRO_NAME == "centos" ] || [ $DISTRO_NAME == "rhel" ] || [ $DISTRO_NAME == "oracle" ]; then
+    if [[ $DISTRO_VERSION =~ ^6\. ]]; then
+        yum -y groupinstall "Development Tools"
+        wget ftp://gnu.mirror.iweb.com/gawk/gawk-5.0.0.tar.xz
```

Comment:
maybe we can make it a common method install_gawk that installs it from repo (ubuntu) and source (rhel based and suse), and get rid of the tool binary gawk

Reply:
> maybe we can make it a common method install_gawk that installs it from repo (ubuntu) and source (rhel based and suse), and get rid of the tool binary gawk

if the package is available for the distro it would be much safer to just install it. compiling requires  dependencies and takes much longer in theory.

Reply:
Package is available but it is not compatible with JSON file created by fio
@lizzha , no change for other distros since we are targeting only 6.x distros